src/components/commons/input/Input.styles.ts

```diff
@@ -0,0 +1,100 @@
+import styled from '@emotion/styled'
+import { css } from '@emotion/react'
+import { color } from '@/src/commons/styles/styles'
+
+const smInputStyles = css`
+  width: 200px;
+  input {
+    font-size: 14px;
+    padding: 15px;
+    &::placeholder {
+      font-size: 14px;
+    }
+  }
+`
+
+const mdInputStyles = css`
+  width: 400px;
+  input {
+    font-size: 16px;
+    padding: 18px 15px;
+    input::placeholder {
+      font-size: 14px;
+    }
+  }
+`
+
+const lgInputStyles = css`
+  input {
+    padding: 20px 15px;
+    &::placeholder {
+      font-size: 14px;
+    }
+  }
+`
+
+const outlinedStyles = css`
+  border: 1px solid #dee2e6;
+  background-color: transparent;
+`
+
+const softStyles = css`
+  background-color: #f5f7fe;
+`
+
+const wrapperStyles = css`
+  padding: 15px;
+  border-radius: 10px;
+  input {
+    padding: 0;
+    border: none;
+    border-radius: 0;
+    &:focus {
+      outline: none;
+    }
+  }
+`
+
+const defaultStyles = css`
+  border: none;
+  background: transparent;
+`
+
+export const InputContainer = styled.div<{ size: string }>`
+  margin: 30px; //삭제필요
+  ${props => (props.size === 'sm' ? smInputStyles : props.size === 'md' ? mdInputStyles : lgInputStyles)}
+`
+
+export const LabelWrapper = styled.div<{ labelType: string; variant: string }>`
+  ${props => (props.variant === 'outlined' ? outlinedStyles : props.variant === 'soft' ? softStyles : '')};
```

Comment:
```suggestion
  ${props => {
    switch (props.variant) {
      case 'outlined':
        return outlinedStyles
      case 'soft':
        return softStyles
      default:
        return ''
    }
  }};
```

저는 이런 식으로 switch case 문을 더욱 선호합니다. 가독성이 더 좋은 것 같아서요. 혜원님은 어떻게 생각하시나요?

Reply:
그리고 같은 로직이 Input에서 발생하는 것으로 보아, 해당 로직을 getVariantStyle 이라는 함수로 만들어서 사용하면 좋을 듯합니다

Reply:
동의합니다 222